user-guide/notebooks/datasets/nldas_time_series.ipynb

```diff
@@ -94,602 +94,2438 @@
    "source": [
```

Comment:
<p>Do we want to change the url text like the other NLDAS notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/235/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>